squidDrone/tests/com/com_message_buffer_tests.cpp

```diff
@@ -0,0 +1,67 @@
+#include "gtest/gtest.h"
+#include "com_message_buffer.hpp"
+
+namespace{
+  // Derived class for testing queue behaviour
+  class ComMessageBufferTest: public com::ComMessageBuffer {
+    public:
+      ComMessageBufferTest(){};
```

Comment:
initialize parent class in the constructor of the child class.
eg. ComMessageBufferTest(): com::ComMessageBuffer(){}

Reply:
Class removed.